tests/config_test.go

```diff
@@ -94,7 +98,9 @@ var _ = Describe("[rfe_id:899][crit:medium][vendor:cnv-qe@redhat.com][level:comp
 			})
 
 			AfterEach(func() {
-				tests.DeleteConfigMap(configMapName, testsuite.GetTestNamespace(nil))
+				if err := kubevirt.Client().CoreV1().ConfigMaps(testsuite.GetTestNamespace(nil)).Delete(context.Background(), configMapName, metav1.DeleteOptions{}); err != nil {
```

Comment:
Would you mind 

```diff
-                               if err := kubevirt.Client().CoreV1().ConfigMaps(testsuite.GetTestNamespace(nil)).Delete(context.Background(), configMapName, metav1.DeleteOptions{}); err != nil {
+                               configMap := kubevirt.Client().CoreV1().ConfigMaps(testsuite.GetTestNamespace(nil))
+                               if err := configMap.Delete(context.Background(), configMapName, metav1.DeleteOptions{}); err != nil {
```

Reduces the line length from 178 to 117. We don't have a defined limit but anything above 150 is a lot imho.

This comments applies to other changes, the problem is that `configMap` is already used in other places. 
Hm. The return type of `ConfigMaps()`  is an interface, so perhaps `configMapIface`? Not sure, I'm terrible with names :)

Reply:
You ignored this comment but it is fine, we don't have any hard requirements for line length anyway.